python/tests/unittests/test_quake_value.py

```diff
@@ -80,6 +80,8 @@ def test_QuakeValueLifetimeAndPrint():
     assert s.count('quake.x') == 2
 
 
+## [SKIP_TEST]
+@pytest.mark.skip(reason="TypeError: unsupported operand type(s) for /: 'float' and 'QuakeValue' on 'kernel3.rx(np.pi / factor[0], qubit3[0])'")
```

Comment:
Fix 
```diff
diff --git a/python/cudaq/kernel/quake_value.py b/python/cudaq/kernel/quake_value.py
index 85c6a7cb6..7b5418108 100644
--- a/python/cudaq/kernel/quake_value.py
+++ b/python/cudaq/kernel/quake_value.py
@@ -138,6 +138,16 @@ class QuakeValue(object):
             return QuakeValue(
                 getattr(arith, opStr)(thisVal, otherVal).result, self.pyKernel)
 
+    def __rtruediv__(self, other):
+        with self.ctx, Location.unknown(), self.pyKernel.insertPoint:
+            thisVal, otherVal, opStr = self.__checkTypesAndCreateQuakeValue(
+                other, 'Div')
+            if opStr == 'DivIOp':
+                opStr = 'DivSIOp'
+
+            return QuakeValue(
+                getattr(arith, opStr)(otherVal, thisVal).result, self.pyKernel)
+    
     def __add__(self, other):
         with self.ctx, Location.unknown(), self.pyKernel.insertPoint:
             thisVal, otherVal, opStr = self.__checkTypesAndCreateQuakeValue(
```

Reply:
Patched in [3deecbb](https://github.com/NVIDIA/cuda-quantum/pull/1077/commits/3deecbb637615bfbef6e30b60e182f920b49ad20)